src/Audit/Audit.php

```diff
@@ -183,9 +183,9 @@ public function log(string $userId, string $event, string $resource, string $use
      * @param int $offset
      * @param Document|null $orderAfter
      *
-     * @return array
+     * @return mixed
      */
-    public function getLogsByUser(string $userId, int $limit = 25, int $offset = 0, Document $orderAfter = null): array
+    public function getLogsByUser(string $userId, int $limit = 25, int $offset = 0, Document $orderAfter = null): mixed
     {
         $result = Authorization::skip(function () use ($userId, $limit, $offset, $orderAfter) {
             $queries = $this->buildQuery(['userId' => $userId], Query::TYPE_EQUAL);
```

Comment:
The typing for `Authorization::skip()` makes this a little weird, but we can do this for now:

```suggestion
    public function getLogsByUser(string $userId, int $limit = 25, int $offset = 0, Document $orderAfter = null): array
    {
        /** @var Document[] $result */
        $result = Authorization::skip(function () use ($userId, $limit, $offset, $orderAfter) {
            $queries = $this->buildQuery(['userId' => $userId], Query::TYPE_EQUAL);
```